clients/admin-ui/cypress/e2e/custom-fields.cy.ts

```diff
@@ -117,19 +117,61 @@ describe("Custom Fields", () => {
       });
     });
 
-    it("can sort", () => {
-      cy.get("tbody > tr")
-        .first()
-        .should("contain", "Taxonomy - Single select");
-      // sort alphabetical
-      cy.getByTestId("column-Label").click();
-      cy.get("tbody > tr").first().should("contain", "Multiple select list");
+    describe("sorting", () => {
+      beforeEach(() => {
+        cy.intercept(
+          "PUT",
+          "/api/v1/plus/custom-metadata/custom-field-definition*",
+          {
+            fixture: "custom-fields/list.json",
+          }
+        ).as("patchCustomFields");
+      });
+      it("should be table to sort", () => {
+        cy.get("tbody > tr")
+          .first()
+          .should("contain", "Taxonomy - Single select");
+        // sort alphabetical
+        cy.getByTestId("column-Label").click();
+        cy.get("tbody > tr").first().should("contain", "Multiple select list");
+
+        // sort reverse
+        cy.getByTestId("column-Label").click();
+        cy.get("tbody > tr")
+          .first()
+          .should("contain", "Taxonomy - Single select");
+      });
 
-      // sort reverse
-      cy.getByTestId("column-Label").click();
-      cy.get("tbody > tr")
-        .first()
-        .should("contain", "Taxonomy - Single select");
+      it("should maintain sort after custom field is enabled/disabled", () => {
```

Comment:
thank you for adding this test!! I was playing around with it and it seems like this test passes even without the change in config in `FidesTable`. the only reason I suspected it might is because it's kind of difficult to get cypress to respond with _updated_ data.

I think what you may have to do is intercept `@getCustomFields` again, after the initial visit, and change its data such that "Taxonomy - Single select" will return as enabled. it's always kind of annoying to do, but something like

```typescript
cy.fixture("custom-fields/list.json").then(customFieldsList => {
  // iterate through customFieldsList and change the value of enabled/disabled for the entry you care about
  const updatedFieldsList = ...
  cy.intercept(
    "GET",
    "/api/v1/plus/custom-metadata/custom-field-definition*",
    {
      body: updatedFieldsList,
    }
  ).as("getCustomFieldsSingleSelectEnabled");
});

// sort, then enable the custom field, and wait for `@patchCustomFields`
cy.wait("@getCustomFieldsSingleSelectEnabled")

```

I haven't tested any of that though, but hopefully we can get the test into a state of failing without your change, then succeeding after the change!

Reply:
@allisonking better late than never :sweat_smile: I believe this is fixed now. I tried turning the config option on and off to see the test pass and fail